crates/sui-json-rpc/src/read_api.rs

```diff
@@ -1107,6 +1107,14 @@ fn convert_to_response(
     let mut response = SuiTransactionResponse::new(cache.digest);
     response.errors = cache.errors;
 
+    if opts.show_raw_input && cache.transaction.is_some() {
+        let sender_signed_data = cache.transaction.as_ref().unwrap().data();
+        match bcs::to_bytes(sender_signed_data) {
+            Ok(t) => response.raw_transaction = t,
```

Comment:
Not about changes in this PR, `SenderSignedData(Vec<SenderSignedTransaction>)`, looks like one sender_signed_data includes a vector of `SenderSignedTransaction`, which maps to a vector of `TransactionData`, is the vector len() possible > 1?
If so, why `SuiTransactionDataV1` always has one transaction kind? also cc @patrickkuo 


The vector length would also affect indexer's handling of txnKind.

Reply:
It might be possible in the future but right now it's always 1 guaranteed by https://github.com/MystenLabs/sui/blob/eeb7c987444dafac0e47b5fc685150cf4a2d5523/crates/sui-types/src/messages.rs#L1596

Reply:
cc @mystenmark to provide more context

Reply:
thanks, yeah feels like if it is not left there for future, maybe we will want to flatten that, but def. not in this PR.